### ISSUE TITLE
__statics problem

### ISSUE THREAD

**Issue by 719media** [quasarframework/quasar#6218]
I'm trying to use statics for an electron build as described on https://quasar.dev/quasar-cli/developing-electron-apps/electron-static-assets

The static assets themselves are copied over fine, which I can see fine by setting `electron.packager.asar: false` in the quasar.conf.js and browsing to the generated electron app.

The app runs just fine in dev mode (`quasar dev -m electron`).

However, the __statics variable does not appear to be working as expected for production (`quasar build -m electron`). I have verified this by throwing an Error in the electron-main.js file (I am unsure how best to troubleshoot/log an error that happens only on electron-main.js and only on a production app build, so I resorted to throwing an Error)

I am not sure how to build a repro of this, given that it's electron, and the codesandbox gateway is down currently. I certainly can try if you tell me this is insufficient information to troubleshoot.

```
if (process.env.PROD) {
  global.__statics = require('path').join(__dirname, 'statics').replace(/\\/g, '\\\\')
}

// new code below
const test = require('path').join(__statics, '/test.json')
const test2 = require('path').join(__dirname, 'statics').replace(/\\/g, '\\\\')
throw Error(`1: ${test} :1 2: ${test2} :2`)
```

On the production build, the Error thrown is

```
1: /Users/macuser/Sites/test-project/dist/electron/Test Project-darwin-x64/Test Project.app/Contents/Resources/app/statics/test.json :1
2: /Users/macuser/Sites/test-project/dist/electron/Test Project-darwin-x64/Test Project.app/Contents/Resources/app/statics :2
```

I was expecting some sort of path without the whole /Users/macuser etc. path, so this is why I am guessing it is not working correctly.

**Comment by 719media** [comment]
I'm going to look into this a bit more before opening

**Comment by 719media** [comment]
Documenting for reference:

So I ran into an issue where I wanted to use `child_process.spawn` with a precompiled asset

The problem was that `spawn` doesn't play nicely with asar files (as noted here: https://github.com/electron/electron/issues/9459). So I set asar: false in the quasar.config.js to address that issue.

It looks like the statics files are copied over into the dist electron directory as 644. So the next problem I ran into was `EACCES`, which was fixed by doing chmod 744 on the file.

I made this part of the build by using `exec('chmod 744 path/to/file')` in the quasar.config.js afterBuild() setting.

Just documenting for future people who are looking for answers for their first time using quasar electron and spawning precompiled asset files included in the package.
